tools/buildPackage.js

```diff
@@ -154,10 +162,31 @@ if (isLinux) {
   cmds.push('copy .\\res\\start-tile-150.png "' + path.join(buildDir, 'resources', 'start-tile-150.png') + '"')
   cmds.push('makensis.exe -DARCH=' + arch + ` res/${channel}/braveDefaults.nsi`)
   cmds.push('ncp ./app/extensions ' + path.join(buildDir, 'resources', 'extensions'))
+
   // Make sure the Brave.exe binary is squirrel aware so we get squirrel events and so that Squirrel doesn't auto create shortcuts.
   cmds.push(`"node_modules/rcedit/bin/rcedit.exe" ./${appName}-win32-` + arch + `/${appName}.exe --set-version-string "SquirrelAwareVersion" "1"`)
 }
 
+// Verify tor binaries and bundle with Brave
+var torPath
+if (isDarwin) {
+  torPath = path.join(buildDir, `${appName}.app`, 'Contents', 'Resources', 'extensions', 'bin')
+} else {
+  torPath = path.join(buildDir, 'resources', 'extensions', 'bin')
+}
+
+cmds.push('mkdirp ' + torPath)
+cmds.push('mkdirp ' + path.join(buildDir, 'resources', 'keys'))
+cmds.push('curl -o ' + path.join(torPath, 'tor') + ' ' + torURL)
+cmds.push('curl -o ' + path.join(torPath, 'tor-sig') + ' ' + torSigURL)
+cmds.push('ncp ./res/keys/ ' + path.join(buildDir, 'resources', 'keys'))
+cmds.push('gpg --import ' + path.join(buildDir, 'resources', 'keys', 'tor-demo-key.pub'))
```

Comment:
if you do `gpg --import ./res/keys/tor-demo-key.pub` then you don't need to copy it into the buildDir

Reply:
also https://github.com/brave/browser-laptop/pull/13550/files#diff-f27081c0d2b1b80f3d8bdaa53cd67021R179 won't be needed